### PR TITLE
Added nullable, clarified desc and support norwegian chars in name

### PR DESCRIPTION
**Pull request by niklud**

### CODE REVIEW

File: docs/swagger.yaml [hunk 1]
```diff
@@ -334,12 +333,13 @@ components:
           description: Organisation number of the merchant for whom the partner is submitting the product order.
         salesUnitName:
           type: string
-          pattern: '^[a-zA-Z0-9- ]{1,30}$'
+          pattern: '^[\p{L}\p{M}0-9-_\s\.]{1,30}$'
```
Comment (cloveras) [inline docs/swagger.yaml:336]: Is intended to not support a name like `Vipps+`?

File: docs/swagger.yaml [hunk 2]
```diff
@@ -359,12 +359,13 @@ components:
...
           description: The bank account number belonging to the organisation that will be used for settlement.
         pricePackageKey:
+          x-nullable: true
           type: string
           pattern: '^[a-zA-Z0-9-]{1,15}$'
           minLength: 1
           maxLength: 255
           example: posstandard
-          description: One of the price packages that is registered for the partner in Vipps system. Please note, it is not the PriceId (3 digit) but the name such as "standard", "price1", etc that is applicable to the partner.
+          description: Optional. One of the price package that is registered for the partner in Vipps system. Please note, it is not the PriceId (3 digit) but the name such as "standard", "price1", etc that is applicable to the partner. If the field is not provided, the merchant has to select it manually in the product order in merchant portal before submitting.
```
Comment (rebeksburnett) [inline docs/swagger.yaml:368]:
```suggestion
          description: Optional. One of the price packages that are registered for the partner in the Vipps system. Please note, it is not the PriceId (3 digit), but the name, such as "standard", "price1", etc, that is applicable to the partner. If the field is not provided, the merchant must select it manually from the product order section of the merchant portal before submitting. 
```

File: docs/swagger.yaml (hunk 1 above)
Comment (niklud) [reply docs/swagger.yaml:336]: I don't really know, I'm open to suggestion, I tried to ensure support for the ones I could think of. Do a suggest change for if you can think of any more.

Comment (cloveras) [inline docs/swagger.yaml:336]:
```suggestion
          pattern: '^[\p{L}\p{M}\d-_\+\s\.]{1,30}$'
```

Comment (cloveras) [reply docs/swagger.yaml:336]: Now accepts `+`. Also changed `0-9` to `\d`.

**Pull request merged**